### DISCUSSION TITLE
Can`t see my brush

### DISCUSSION THREAD

**Discussion by Shawn-Akii** [spimort/TerraBrush#175] [Q&A]
According to tutorial,when I get started,I should see a brush circle on map,but now I can see only cursor instead of brush area,does anyone know how to turn brush area on?tutorial is 4.2, but this addon is 4.3,many things change maybe...

**Comment by spimort** [comment]
Humm, I've never seen that, are you sure the plugin is enabled? 🙂

**Comment by MartinDanihelDev** [comment]
Try to switch to Forward+ if possible. I experienced this when I had compatibility mode.
